### PR TITLE
Modify JVM Crypto rules: Change framework name

### PR DESCRIPTION
**Pull request by loris-s-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by daniel-teuchert-sonarsource — CHANGES_REQUESTED** [review]
Overall LGTM. Maybe you should also adjust the file names since they also contain the typo?

File: docs/header_names/allowed_framework_names.adoc [hunk 1]
```diff
@@ -36,7 +36,7 @@
 * FasterXML
 * Gson
 * Android
-* Java Cryptographic Extension
```
Comment (egon-okerman-sonarsource) [inline docs/header_names/allowed_framework_names.adoc:39]: It's in there twice now, remove one

**Review by egon-okerman-sonarsource — APPROVED** [review]
Needs a small change, afterwards good to merge

File: docs/header_names/allowed_framework_names.adoc [hunk 2]
```diff
@@ -36,7 +36,7 @@
...
+* Java Cryptography Extension
```
Comment (loris-s-sonarsource) [inline docs/header_names/allowed_framework_names.adoc:39]:
```suggestion
```

File: docs/header_names/allowed_framework_names.adoc (hunk 1 above)
Comment (loris-s-sonarsource) [reply docs/header_names/allowed_framework_names.adoc:39]: good catch!

**Comment by loris-s-sonarsource** [comment]
@daniel-teuchert-sonarsource good catch! this is done

**Review by daniel-teuchert-sonarsource — CHANGES_REQUESTED** [review]
After changing the file names you will also have to adjust the include statements (e.g. include::how-to-fix-it/java-cryptographic-extension.adoc[] )

**Pull request merged**